fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -789,6 +789,7 @@ protected void addResourceHttpHeaders(final FedoraResource resource, final boole
             servletResponse.addHeader(LINK, "<" + canonical + ">;rel=\"canonical\"");
 
         }
+        addRegisterHeaders();
```

Comment:
This (`addResourceHttpHeaders`) only adds the headers to requests that return a resource body, so a `GET` request (in most cases) and `HEAD`. So `DELETE`, `PATCH`, `OPTIONS`, `POST` and `PUT` requests don't get them.

Also no requests for `fcr:acl`, `fcr:versions`, `fcr:fixity` and `fcr:tx`. These are handled by other classes in fcrepo-http-api (i.e. `FedoraAcl`, `FedoraTombstones`, `FedoraFixity`, etc).